common/kubernetes/configs.go

```diff
@@ -120,9 +120,6 @@ func GetKubeClientConfig(uri *url.URL) (*kube_client.Config, error) {
 	if len(kubeConfig.Host) == 0 {
 		return nil, fmt.Errorf("invalid kubernetes master url specified")
 	}
-	if len(kubeConfig.GroupVersion.Version) == 0 {
```

Comment:
Thanks @caesarxuchao 

To give you more context this started failing after bumping the Kubernetes version in #1158 from https://github.com/kubernetes/kubernetes/commit/644d651c690774376112cd3f257af0308e95a392 (~late Feb 2016) to https://github.com/kubernetes/kubernetes/commit/4bb30e00978219640215ac04e84caf5bf398be92 (~mid May 2016). After that NewNonInteractiveDeferredLoadingClientConfig function started returning empty version.